physionet-django/physionet/context_processors.py

```diff
@@ -20,3 +20,6 @@ def platform_config(request):
         'STRAPLINE': settings.STRAPLINE,
         'SOURCE_CODE_REPOSITORY_LINK': settings.SOURCE_CODE_REPOSITORY_LINK
     }
+
+def session_security_enabled(request):
```

Comment:
```suggestion
def session_security_config(request):
```
Makes more sense semantically + it's more consistent with the other context processors